t/app/helpers/send_email.t

```diff
@@ -42,7 +42,7 @@ my $email = Email::MIME->new($email_as_string);
 my $expected_email_content =   path(__FILE__)->parent->child('send_email_sample.txt')->slurp;
 my $name = FixMyStreet->config('CONTACT_NAME');
 $name = "\"$name\"" if $name =~ / /;
-my $sender = $name . ' <' . FixMyStreet->config('DO_NOT_REPLY_EMAIL') . '>';
+my $sender = '"' . $name . '" <' . FixMyStreet->config('DO_NOT_REPLY_EMAIL') . '>';
```

Comment:
Ah, in the test CONTACT_NAME will be "FixMyStreet" by default (it uses the example general.yml file), so this never did anything; it would indeed fail if it had a space in it (it remained with one set of quotes).